src/components/CompareResults/ResultsTable.tsx

```diff
@@ -36,13 +36,25 @@ const cellsConfiguration: CompareResultsTableConfig[] = [
   },
   {
     key: 'comparisonSign',
-
     gridWidth: '0.2fr',
   },
   {
     name: 'New',
     key: 'new',
-
+    gridWidth: '1fr',
+  },
+  {
+    name: 'Base`σ`',
+    key: 'basestandarddeviation',
+    gridWidth: '1fr',
+  },
+  {
+    key: 'comparisonSignstandarddeviation',
+    gridWidth: '0.2fr',
+  },
+  {
+    name: 'New`σ`',
+    key: 'newstandarddeviation',
```

Comment:
Can you use camelCase for the naming of the keys.

Reply:
definitely